symantec-messaging-gateway/info.json

```diff
@@ -1,349 +1,347 @@
-                        "Logical IP"
-                    ],
-                    "name": "mandatoryFilterId",
-                    "value": "Subject",
-                    "tooltip": "Select search by",
-                    "description": "Select search by."
-                },
-                {
-                    "title": "Search Value",
-                    "required": true,
-                    "editable": true,
-                    "visible": true,
-                    "type": "text",
-                    "name": "mandatoryFilterValue",
-                    "value": "",
-                    "tooltip": "Value to search for based on the search by type. Exp baduser@badomain.com for Search by Sender",
-                    "description": "Value to search for based on the search by type. Exp baduser@badomain.com for Search by Sender"
-                },
-                {
-                    "title": "Start Time",
-                    "required": true,
-                    "editable": true,
-                    "visible": true,
-                    "type": "datetime",
-                    "name": "start_time",
-                    "value": "",
-                    "tooltip": "Search start time (Search from:)",
-                    "description": "Search start time (Search from:)"
-                },
-                {
-                    "title": "End Time",
-                    "required": true,
-                    "editable": true,
-                    "visible": true,
-                    "type": "datetime",
-                    "name": "end_time",
-                    "value": "",
-                    "tooltip": "Search end time (Search until:)",
-                    "description": "Search end time (Search until:)"
-                },
-                {
-                    "title": "Limit",
-                    "required": false,
-                    "editable": true,
-                    "visible": true,
-                    "type": "integer",
-                    "name": "entriesPerPage",
-                    "value": 15,
-                    "tooltip": "Entries to return per page",
-                    "description": "Entries to return per page"
-                },
-                {
-                    "title": "Offset",
-                    "required": false,
-                    "editable": true,
-                    "visible": true,
-                    "type": "integer",
-                    "name": "pageNumber",
-                    "value": 1,
-                    "tooltip": "Page number",
-                    "description": "Page number"
-                }
-            ],
-            "enabled": true
+          "title": "IP Address",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "ip"
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "audit_logs_search",
+      "title": "Quick Audit Log Search",
```

Comment:
why name given as quick audit log search? what is quick signifies here?

Reply:
It is submitted by community.